WordPress/metadata/release_notes.txt

```diff
@@ -1,3 +1,7 @@
-This release is short and sweet: after deleting a site, you might’ve noticed that same site sticking around in your My Site list until you restarted the app.
```

Comment:
I manually removed the following entry that was automatically extracted from the `RELEASE-NOTES.txt`, given that it was marked `[internal]`:

```
* [*] [internal] Site creation: Adds a new screen asking the user the name of the site [https://github.com/wordpress-mobile/WordPress-Android/pull/16259]
```
@mkevins I believe you marked it as `[internal]` because, from what I read from your associated PR, this was behind a feature flag, so that's not yet visible to end users, right? Just wanted to confirm that me removing this item from what will end up in the 19.7 release notes was appropriate 🙂

Reply:
Sort of.. this is actually enabled (feature flag default `true`), but is also conditioned with an ExPlat flag which is used remotely to run an experiment (upcoming). The internal note indicates that this should be included in calls for testing, but not in the public release notes (more info on the "internal" label in this Slack thread: p1649059586421809-slack-C033D5RCN2Y ).

tl;dr removing those from the public release notes is correct :+1:  Thank you for double checking!

Reply:
Got it, thanks! (And TIL about the meaning we give to `[internal]` in that context!)

cc @thehenrybyrd for info about still including that item in the CfT for WPAndroid 19.7 despite me removing it from the draft for public facing release notes here.

Reply:
@AliSoftware Yep! `[internal]` has been used for that in WPiOS for a while, we're just starting to use in in WPAndroid - I've got a p2 post drafted about it and other release notes best practices, should be up this week :)